### PR TITLE
SEO-2393 - ReactJS image alt text changes

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @Aishwarya-Ganesan 
Please review the image alt text changes for ReactJS.

Regards,
Sharly A.

### CODE REVIEW

**Comment by Sharly-Atieno** [comment]
rebuild

File: ReactJS/Chart/Chart-Dimensions.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ You can also set the chart dimension by using the [`size`](../api/ejchart#member
 
 {% endhighlight %}
 
-![](/js/Chart/Chart-Dimensions_images/Chart-Dimensions_img1.png)
+![ReactJS Chart dimentions](Chart-Dimensions_images/Chart-Dimensions_img1.png)
```
Comment (Aishwarya-Ganesan) [inline ReactJS/Chart/Chart-Dimensions.md:50]: Check dimensions spelling mistake

File: ReactJS/CircularGauge/Getting-Started.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-# Getting Started
+# Getting Started with ReactJS CircularGauge
 
 * This section encompasses how to configure Circular Gauge. You can provide data to a Circular Gauge and make them to display in a required way.
 
 * The following screen shot displays a Circular Gauge, which visually represents the functions of an Automobile speedometer with RPM (Rotation per Minute), kph (Kilometer per hour) and it denotes the speed level indicators (Safe, Caution and Danger).
 
 
-![](Getting-Started_images/Getting-Started_img11.png)
+![ReactJS CirculaGauge Getting Started](Getting-Started_images/Getting-Started_img11.png)
```
Comment (Aishwarya-Ganesan) [inline ReactJS/CircularGauge/Getting-Started.md:17]: CirculaGauge - Spelling mistake. Correct this in below all places.

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Commented.

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are Fine.

**Pull request merged**